go/py client intermittent CI failures

To my knowledge, this only occurs in CI, and only when a full `check` run takes place, it isn't sufficient to run `:go:check` or `:py-client:check` to reproduce consistently enough to rely on results:
`py-client:check` in CI failed 0 / 96 tries
`go:check` in CI failed 6 / 96 tries
`check` failed 11 / 32 tries

Failure messages in go:
https://github.com/niloc132/deephaven-core/actions/runs/11668272387/job/32487558122#step:10:4368
```
=== RUN   Example_fetchTable
--- FAIL: Example_fetchTable (0.35s)
got:
error when opening table: rpc error: code = Unavailable desc = error reading from server: EOF
Successfully opened the old table!
want:
Successfully opened the old table!
```

https://github.com/niloc132/deephaven-core/actions/runs/11668272387/job/32487562349#step:10:4174
```
=== RUN   TestOpenTable
    test_tools.go:67: OpenTable error rpc error: code = Unavailable desc = error reading from server: EOF
--- FAIL: TestOpenTable (0.02s)
```

https://github.com/niloc132/deephaven-core/actions/runs/11668272387/job/32487563983#step:10:4376
```
 === RUN   Example_runScript
--- FAIL: Example_runScript (0.36s)
got:
error when opening table: rpc error: code = Unavailable desc = error reading from server: EOF
want:
Got table snapshot!
It has 5 rows and 2 columns
```

https://github.com/niloc132/deephaven-core/actions/runs/11668272387/job/32487564477#step:10:3958
```
=== RUN   TestFieldSync
    test_tools.go:67: OpenTable error rpc error: code = Unavailable desc = error reading from server: EOF
--- FAIL: TestFieldSync (0.06s)
```

https://github.com/niloc132/deephaven-core/actions/runs/11668272387/job/32487565020#step:10:4174
```
=== RUN   TestFieldSync
    test_tools.go:67: OpenTable error rpc error: code = Unavailable desc = error reading from server: EOF
--- FAIL: TestFieldSync (0.09s)
```

https://github.com/deephaven/deephaven-core/pull/6318 was an attempt at mitigating this, commenting out `Example_runScript`, but the others still seem to happen. After that PR, we've seen both
```
 === RUN   TestFieldSync
    test_tools.go:67: OpenTable error rpc error: code = Unavailable desc = error reading from server: EOF
--- FAIL: TestFieldSync (0.08s)
```
and
```
 === RUN   TestOpenTable
    test_tools.go:67: OpenTable error rpc error: code = Unavailable desc = error reading from server: EOF
--- FAIL: TestOpenTable (0.02s)
```

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -50,7 +50,7 @@ javax-inject = "1"
 javax-validation = "2.0.1.Final"
 jdom2 = "2.0.6.1"
 jetbrains = "26.0.1"
-jetty = "11.0.24"
+jetty = "11.0.20"
 jpy = "0.18.0"
 jsinterop = "2.0.2"
 # google is annoying, and have different versions released for the same groupId

